packages/jest-emotion/src/matchers.js

```diff
@@ -35,13 +49,35 @@ function valueMatches(declaration, value) {
   return value === declaration.value
 }
 
-function toHaveStyleRule(received: *, property: *, value: *) {
+function toHaveStyleRule(
+  received: *,
+  property: *,
+  value: *,
+  options?: { target?: string, media?: string } = {}
+) {
+  const { target = '', media = '' } = options
```

Comment:
There's no need for the default values here, leave them as undefined and handle undefined instead of an empty string where necessary.